src/Text/Pandoc/Class/PandocMonad.hs

```diff
@@ -574,11 +574,25 @@ getDefaultReferencePptx = do
      Nothing   -> foldr addEntryToArchive emptyArchive <$>
                      mapM pathToEntry paths
 
--- | Read file from user data directory or,
--- if not found there, from the default data files.
+-- | Checks if the file path is relative to a parent directory.
+isRelativeToParentDir :: FilePath -> Bool
+isRelativeToParentDir fname =
+  let canonical = makeCanonical fname
+   in length canonical >= 2 && take 2 canonical == ".."
+
+-- | Returns user data directory if it exists and if the file path exists
+-- within it.
```

Comment:
This comment might be a bit misleading, since we aren't really checking for the existence of the file path.
Not sure how to word it better, though.

Reply:
Hm. Maybe that says something about the function itself. Trying something else, what do you think?